contentrepo/settings/base.py

```diff
@@ -145,6 +145,7 @@
 MEDIA_ROOT = os.path.join(BASE_DIR, "media")
 MEDIA_URL = "/media/"
 
+TEST_STATIC_ROOT = os.path.join(BASE_DIR, "home/tests/test_static")
```

Comment:
Do we need this? If we just do the relative path `open("home/tests/test_static/test.jpeg", ...`, does that not work just fine? Happy to keep this here if that doesn't work.